test/AlgebraicGeometry/ToricVarieties/line_bundle_cohomologies.jl

```diff
@@ -68,6 +68,5 @@
     
     @testset "Sections of anticanonical bundle" begin
         @test all_cohomologies(l4) == [4551, 0, 0, 0, 0, 0]
```

Comment:
```suggestion
        @test all_cohomologies(l4) == [4551, 0, 0, 0, 0, 0]
        # test disabled due to memory requirements, see #2441
        @test_skip length(basis_of_global_sections(l4)) == 4551
```

Maybe change it to `@test_skip` instead of removing the test so that we remember to enable this again when the code gets improved?